seafile/README.md

```diff
@@ -0,0 +1,9 @@
+# Seafile
+
+This is an example configuration on how to use [Seafile](https://www.seafile.com/) with Caddy.
+
+To make Seafile work with Caddy ensure to have two separate virtual hosts configured in your Caddyfile.
+Combining the two proxy directives under a single virtual host entry will cause problems when uploading
```

Comment:
With Caddy, I'm trying to avoid the use of the term "virtual host" since the concept doesn't translate over exactly. Instead, we just call them "sites". In a sense, you've actually got 2 sites configured with your Caddyfile (but they are on the same host -- see what I mean?).